shared/src/reducers/fadersReducer.ts

```diff
@@ -280,41 +280,42 @@ export const faders = (
             })
             return nextState
         case FADER_ACTIONS.SET_ASSIGNED_CHANNEL:
+            let newAssignment: IChannelReference[] =

```

Comment:
`newAssignment` is an array so it would be nice that it was named in plural, `newAssignments`, to indicate this fact

Reply:
renamed